src/main/java/fiji/threshold/Auto_Threshold.java

```diff
@@ -175,7 +175,7 @@ public void run(String arg) {
 							imp3 = new ImagePlus("Auto Threshold", stackNew);
 							imp3.updateAndDraw();
 							MontageMaker mm= new MontageMaker();
-							mm.makeMontage( imp3, 4, 4, 1.0, 1, (ml-1), 1, 0, true); // 4 columns and 4 rows
+							mm.makeMontage( imp3, 5, 4, 1.0, 1, (ml-1), 1, 0, true); // 4 columns and 4 rows
```

Comment:
@landinig This argument 4 changed to 5, but the comment still says "4 columns and 4 rows". Is that correct?

Reply:
Oops, the 5 is correct but the comment was not updated. Will do now., thanks for spotting this.